presto-docs/src/main/sphinx/functions/geospatial.rst

```diff
@@ -58,6 +58,17 @@ Constructors
 
     Returns a geometry type polygon object from WKT representation.
 
+.. function:: ST_SphericalGeometryFromGeometry(Geometry) -> SphericalGeometry
+
+    The SphericalGeometry type supports the same set of shapes as Geometry, e.g. point, multipoint, linestring, multilinestring, polygon, multipolygon, etc., but assumes they're
+    shapes on the spherical surface of earth, following the WGS84 reference system (http://www.unoosa.org/pdf/icg/2012/template/WGS_84.pdf). This function takes a Geometry object,
```

Comment:
s/earth/the Earth

Reply:
If it’s spherical then it can’t be WGS84, which defines a model based on a spheroid.

Reply:
@martint you mean a ellipsoid? thanks for pointing out; will correct.

Reply:
A spheroid is a special form of ellipsoid. All spheroids are ellipsoids, but not the other way around. WGS84 uses an “oblate spheroid”